lib/ReactViews/StandardUserInterface/StandardUserInterface.jsx

```diff
@@ -50,7 +51,7 @@ const StandardUserInterface = React.createClass({
             that.acceptDragDropFile();
         };
 
-        document.addEventListener('dragover', this.dragOverListener, false);
+        document.getElementById('ui').addEventListener('dragover', this.dragOverListener, false);
```

Comment:
This is configurable by the implementation of TerriaJS, could we change it to using a `ref` of the outermost element?